### PR TITLE
Add note about not using apt/dnf searches to AMLFS Client install docs

### PR DESCRIPTION
**Pull request by t-mialve**
Adding a note clarifying that doing an `apt/dnf search` will not work.

Ideally* it'll add another note box below this with the new text
image

If someone from the docs team could add a screenshot from the staging environment confirming the behavior that would be great!

### CODE REVIEW

**Comment by Jak-MS** [comment]
@pauljewellmsft 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by pauljewellmsft** [comment]
#sign-off

File: azure-managed-lustre/includes/client-install-version-rhel-7.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ sudo yum install amlfs-lustre-client-2.15.4_42_gd6d405d-$(uname -r | sed -e "s/\
 
 > [!NOTE]
 > The metapackage version does not always align with the kernel version. Use the install command above to install the proper metapackage.
+> [!NOTE]
+> Running a `yum search amlfs-lustre-client` does not show all available packages for your distro. To see all available `amlfs-lustre-client` packages, please run `yum list --showduplicates "amlfs-lustre-client*"`.
```
Comment (Court72) [inline azure-managed-lustre/includes/client-install-version-rhel-7.md:18]:
```suggestion

> [!NOTE]
> Running a `yum search amlfs-lustre-client` doesn't show all available packages for your distro. To see all available `amlfs-lustre-client` packages, run `yum list --showduplicates "amlfs-lustre-client*"`.
```

File: azure-managed-lustre/includes/client-install-version-rhel-8-9.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ sudo dnf install amlfs-lustre-client-2.15.4_42_gd6d405d-$(uname -r | sed -e "s/\
 
 > [!NOTE]
 > The metapackage version does not always align with the kernel version. Use the install command above to install the proper metapackage.
+> [!NOTE]
+> Running a `dnf search amlfs-lustre-client` does not show all available packages for your distro. To see all available `amlfs-lustre-client` packages, please run `dnf list --showduplicates "amlfs-lustre-client*"`.
```
Comment (Court72) [inline azure-managed-lustre/includes/client-install-version-rhel-8-9.md:18]:
```suggestion

> [!NOTE]
> Running a `dnf search amlfs-lustre-client` doesn't show all available packages for your distro. To see all available `amlfs-lustre-client` packages, run `dnf list --showduplicates "amlfs-lustre-client*"`.
```

File: azure-managed-lustre/includes/client-install-version-ubuntu.md [hunk 3]
```diff
@@ -14,6 +14,8 @@ sudo apt install amlfs-lustre-client-2.15.4-42-gd6d405d=$(uname -r)
 
 > [!NOTE]
 > This command installs a metapackage that will keep the version of Lustre aligned with the installed kernel. In order for this to work, you must use `apt full-upgrade` instead of simply `apt upgrade` when updating your system.
+> [!NOTE]
+> Running a `apt search amlfs-lustre-client` does not show all available packages for your distro. To see all available `amlfs-lustre-client` packages, please run `apt list -a "amlfs-lustre-client*"`.
```
Comment (Court72) [inline azure-managed-lustre/includes/client-install-version-ubuntu.md:18]:
```suggestion

> [!NOTE]
> Running an `apt search amlfs-lustre-client` doesn't show all available packages for your distro. To see all available `amlfs-lustre-client` packages, run `apt list -a "amlfs-lustre-client*"`.
```

**Pull request merged**